cmd/titus-storage/ebs.go

```diff
@@ -0,0 +1,312 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/Netflix/titus-executor/logger"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/credentials/ec2rolecreds"
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/mvisonneau/go-ebsnvme/pkg/ebsnvme"
+)
+
+func ebsRunner(ctx context.Context, command string, config MountConfig) error {
+	if config.ebsVolumeID == "" {
+		return fmt.Errorf("TITUS_EBS_VOLUME_ID unset, unable to perform any EBS-related operations")
```

Comment:
This could be an errors.New

Reply:
I have to re-learn what is best practice here every time for some reason.

I thought fmt.errorf (+ and optional %w to wrap another error) was considered best practice?
https://blog.golang.org/go1.13-errors

Would you like me to replace all the fmt.errorfs with errors.New? Or just this one instance?